spec/requests/ticket_desk_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "Ticket Desks"
+  describe "Fetch ticket desks" do
+    it 'works and return status 201' do
```

Comment:
The endpoint returns status 204 - no content. Maybe add ticket_desk record and try to get 200.